python/lsst/analysis/tools/__init__.py

```diff
@@ -21,19 +21,20 @@
 r"""This packages contains various analysis tools and PipelineTasks which
 run them.
 
-There are two types of primary tools within this package; `AnalysisMetrics
-and `AnalysisPlots`, both of which are specialized subclasses of `AnalysisTool`.
+There are two primary types of tools within this package: `AnalysisMetrics` and
+`AnalysisPlots`, both of which are specialized subclasses of `AnalysisTool`.
 
-`AnalysisTool`\ s are objects that have three stages; prep, process, and
-post-process. Creating a new `AnalysisTool` is the process of choosing (though
-configuration) what `AnalysisAction` will run for each of those stages.
+An `AnalysisTool` object has three stages: prep, process, and produce.
+Creating a new `AnalysisTool` involves choosing (through
+configuration) an `AnalysisAction` to run for each of those stages.
```

Comment:
Since you can use multiple actions for each stage, how about "which `AnalysisAction`s to run..."

Reply:
Modified the final sentence to read:
> Creating a new `AnalysisTool` involves choosing (through configuration) one or
> more `AnalysisAction` classes to run for each of those stages.